libs/pdi-scanner/src/ts/scanner_client.ts

```diff
@@ -454,7 +454,13 @@ export function createPdiScannerClient() {
      * Disconnects pdictl from the scanner, but keeps it running.
      */
     async disconnect(): Promise<SimpleResult> {
-      return sendSimpleCommand({ command: 'disconnect' });
+      const result = await sendSimpleCommand({ command: 'disconnect' });
+
+      if (result.err()?.code === 'disconnected') {
```

Comment:
Note that this change will affect production as well. Essentially, it makes "error: disconnected" be an acceptable answer when we're trying to disconnect.